### ISSUE TITLE
ss-border does nothing

### ISSUE THREAD

**Issue by Nazdro** [andrewgioia/keyrune#62]
When adding the ss-border (to get the white border around the set symbol) the i:after does not get resized to fit the glyph. It stays as a 0x0 px object, without modifying the symbol.

**Comment by andrewgioia** [comment]
It currently only works in Webkit browsers and even then only when `.ss-grad` is not being used. It's not ideal and was never really fully baked.

<img width="63" alt="schermata 2018-02-14 alle 9 59 18 pm" src="https://user-images.githubusercontent.com/221473/36238714-52d5880e-11d2-11e8-8f56-7fd59f6867b5.png">

I haven't taken a look at it in a long time though and there's perhaps a much better way to handle it automatically, I can certainly start taking another crack at it.

**Comment by Nazdro** [comment]
Good to know.

I needed the symbols to stand out from a dark background, which I solved using:
```
.ss-border::before{
  text-shadow:
  -6px -6px 10px #fff,
  6px -6px 10px #fff,
  -6px  6px 10px #fff,
  6px  6px 10px #fff;
}
```
It's not perfect but without the blur-radius, all the jagged edges of the vectors don't get shaded. Not sure if it'll help you, but extra info shouldn't hurt.

**Comment by andrewgioia** [comment]
Cool thanks for the tip. Just briefly looking at it, `-webkit-text-stroke: .05em #fff;` on the `:before` also seems to do what I was trying originally but is only for Webkit browsers still.

![schermata 2018-02-15 alle 1 49 01 pm](https://user-images.githubusercontent.com/221473/36274843-0928e7ca-1257-11e8-9201-d454e7137b83.png)

It at least works for both the gradient and non-gradient versions though.

**Comment by andrewgioia** [comment]
Resolved as best as possible in v3.16.1 and will be fully handled in v4.
